### PR TITLE
[MIRROR] Adds Anti-Kudzu ERT

### PR DESCRIPTION
**Pull request by SinguloBot**
Original PR: https://github.com/BeeStation/BeeStation-Hornet/pull/5484
--------------------

## About The Pull Request
I have seen multiple occasions of kudzu getting way out of hand, and an ERT sent to fight it, which is generally hastily patched together and ill-equipped. This formalizes the Weed Whacker squad.

## Why It's Good For The Game
Admins can now ICly control extremely out of hand kudzu spread.

## Changelog
:cl:
add: The Weed Whacker ERT is now available to combat extreme kudzu infestations.
/:cl:

### CODE REVIEW

File: code/game/objects/items/miscellaneous.dm [hunk 1]
```diff
@@ -376,3 +376,8 @@
 	name = "living lube delivery beacon"
 	default_name = "Offensive"
 	mob_choice = /mob/living/simple_animal/hostile/retaliate/clown/lube
+
+/obj/item/choice_beacon/pet/goat
+	name = "goat delivery beacon"
+	default_name = "Billy"
+	mob_choice = /mob/living/simple_animal/hostile/retaliate/goat
```
Comment (KubeRoot) [inline code/game/objects/items/miscellaneous.dm:383]: Oh god, *they get a goat delivery beacon*

File: code/modules/clothing/outfits/ert.dm [hunk 2]
```diff
@@ -293,6 +293,33 @@
...
+	glasses = /obj/item/clothing/glasses/night
+	back = /obj/item/storage/backpack
+	belt = /obj/item/storage/belt/janitor/full
+	r_pocket = /obj/item/grenade/chem_grenade/antiweed
+	l_pocket = /obj/item/grenade/chem_grenade/antiweed
+	l_hand = /obj/item/scythe
+	backpack_contents = list(/obj/item/storage/box/engineer=1,
+		/obj/item/storage/box/lights/mixed=1,
+		/obj/item/melee/baton/loaded=1,
+		/obj/item/choice_beacon/pet/goat,
```
Comment (KubeRoot) [inline code/modules/clothing/outfits/ert.dm:310]: No, wait, *they get a goat delivery beacon* ***each?***

**Review by KubeRoot — APPROVED** [review]
Looks good

**Comment by Laboredih123** [comment]
I doubt we will use this but why not, merging

File: code/game/objects/items/miscellaneous.dm (hunk 1 above)
Comment (Laboredih123) [reply code/game/objects/items/miscellaneous.dm:383]: humour

File: code/modules/clothing/outfits/ert.dm (hunk 2 above)
Comment (Laboredih123) [reply code/modules/clothing/outfits/ert.dm:310]: humour multiplied

**Pull request merged**